support-workers/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "support-workers",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --group=-integration",
+    "it-test": "jest --group=integration",
+    "lint": "eslint src/typescript/**/*.ts",
+    "check-formatting": "prettier --check **.ts",
+    "build": "tsc",
+    "clean-target": "rm -rf target && mkdir -p target",
+    "install-runtime-dependencies": "yarn install --modules-folder target/typescript/node_modules --production",
```

Comment:
This line installs only the runtime (non-dev) dependencies into the target directory so they can be zipped up into the deployment artifact